src/components/organisms/AdminVenues/AdminVenues.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useMemo } from "react";
+import { Button } from "react-bootstrap";
+import { Link } from "react-router-dom";
+
+import { AnyVenue, VenueTemplate } from "types/venues";
+
+import { WithId } from "utils/id";
+
+import "./AdminVenues.scss";
+
+interface AdminVenuesProps {
+  venues: WithId<AnyVenue>[];
+}
+
+export const AdminVenues: React.FC<AdminVenuesProps> = ({ venues }) => {
+  const partyVenues = useMemo(() => {
+    return venues?.filter((venue) => venue.template === VenueTemplate.partymap);
+  }, [venues]);
```

Comment:
```suggestion
  const partyVenues = useMemo(() => venues?.filter((venue) => venue.template === VenueTemplate.partymap) ?? [], [venues]);
```

Reply:
Venues is required array field. No need for this.

Reply:
The inline syntax looks good though.

Reply:
Suggestion was based on the fact that it was using `venues?.`, implying it could be undefined.